bibliopixel/animation/animation.py

```diff
@@ -7,19 +7,28 @@
 class BaseAnimation(object):
     free_run = False
 
-    def __init__(self, led):
-        self._led = led
+    def __init__(self, layout):
+        self._layout = layout
         self.internal_delay = None
 
+    @property
+    def _led(self):
+        """Many BiblioPixelAnimations use the "protected" variable _led."""
```

Comment:
By many, you mean ALL of them? That's literally the only way to set your pixels.
Thank you for at least adding this in... 2 things we need to do though:
- BPA should switch *every* anim over to _layout
- Please update the wiki doc ASAP to show _layout once this is merged in.